pom.xml

```diff
@@ -22,6 +22,7 @@
     <module>${project.name}-product/products/msgraph-calendar</module>
     <module>${project.name}-product/products/msgraph-todo</module>
     <module>${project.name}-product/products/msgraph-mail</module>
+    <module>${project.name}-product/products/msgraph-chat</module>
```

Comment:
I would change the names to msgraph-teams.

Reply:
had it like that, then changed it back to chat to streamline it with existing products here. (see todo, mail, calendar)
... microsoft does just the same: their internal api-docs are referring to /chat and also the endpoints are called like that. but the user-docs still call the target-product to use chats MS Teams ;)